### PR TITLE
Made CLI and macro options consistent

### PR DESCRIPTION
**Pull request by alexreg**
Also updated docs to be more readable and reflect changes in options.

---
This change is [Reviewable](https://reviewable.io/reviews/crabtw/rust-bindgen/324)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,29 +93,30 @@ Examples
 
 *or*
 
-Cargo.toml
+`Cargo.toml`
 
-    [dependencies.bindgen]
-    git = "https://github.com/crabtw/rust-bindgen.git"
+    [dependencies]
+    bindgen = ">= 0"
```
Comment (Yamakaky) [inline README.md]: shoud be 0.16 or *

Comment (alexreg) [inline README.md]: I don't think so. Certainly not the latter, since it's not allowed on crates.io any more. The former would break too quickly,

**Comment by Yamakaky** [comment]
Please rebase

**Comment by alexreg** [comment]
Rebasing is something the maintainer needs to do.

**Comment by Yamakaky** [comment]
Well, I think it should be you since the merge has conflicts, but whatever, I'll do it.

**Comment by alexreg** [comment]
It had no conflicts when I opened the PR. :) I can rebase if you like, but I think the resolution will be relatively easy. Thanks.

**Comment by Yamakaky** [comment]
Merged, thanks

**Pull request closed without merging**